root_VS2010/programs/C#/Frameworks/Infrastructure/Framework/Util/FxHttpSessionIndex.cs

```diff
@@ -59,6 +60,9 @@ public static class FxHttpSessionIndex
         /// <summary>ボタン履歴を保持する「キュー」を格納するSessionキー</summary>
         public const string BUTTON_HISTORY = "ButtonHistory";
 
+        /// <summary> Session key that contains the dummy value. </summary>
+        public const string DUMMY = "dummy";
```

Comment:
@yusukemaegawa 

There is no problem. This has been synchronized for merge tasks.
Future, deny to add DUMMY property to the other old applications by this.